### ISSUE TITLE
GeoMx module

### ISSUE THREAD

**Issue by alanhoyle** [ewels/MultiQC#1801]
### Name of the tool

geomxngspipeline

### Tool homepage

https://nanostring.com/products/geomx-digital-spatial-profiler/geomx-dsp-overview/Total reads: 17,674,461 Quality trimmed: 35,348,922 (200.00 %) Adapter trimmed: 0 (0.00 %) Hard trimmed: 0 (0.00 %) Dropped after trim: 100,166 (0.57 %) Remaining after trim: 17,574,295 (99.43 %)  Rescued by stitching: 2,120 (0.01 %) Dropped after stitching: 85,411 (0.48 %) Total after stitching: 17,491,004 (98.96 %)  Total reads matched: 17,181,177 (98.23 %) Matched with HD0: 16,899,775 (96.62 %) Matched with HD1: 279,585 (1.60 %) Matched with 11bp barcode: 0 (0.00 %) Matched by UMI: 3,612 (0.02 %) Dropped as not matched: 307,707 (1.76 %) UMI Q30: 0.9898 RTS Q30: 0.9900  Total reads dropped: 493,284 (2.79 %) Reads after dedup: 2,839,445  Dedup time: 21,998 ms Total execution time: 327,308 ms

### Tool description

The GeoMx Digital Spatial Profiler (DSP) provides morphological context in spatial transcriptomics and spatial proteomics experiments from just one slide.

### Tool output

[Archive.zip](https://github.com/ewels/MultiQC/files/10016108/Archive.zip)

### Data suitable for MultiQC plot(s)

basically all of the metrics except the Dedup time and Total execution time are suitable

### Most interesting data for the General Stats table

Total reads matched
Total reads dropped
Reads after dedup
